pennylane/operation.py

```diff
@@ -1242,7 +1242,7 @@ def matrix(self, wire_order=None):
         return expand_matrix(canonical_matrix, wires=self.wires, wire_order=wire_order)
 
     def eigvals(self):
-        op_eigvals = self.compute_eigvals(*self.parameters, **self.hyperparameters)
+        op_eigvals = super().eigvals()
```

Comment:
Ehm...this was a serious bug I overlooked before, and which turned up when running docstring examples...

Reply:
oops! We should add a test to catch it :)